vms/avm/txs/parser.go

```diff
@@ -157,19 +146,3 @@ func parse(cm codec.Manager, signedBytes []byte) (*Tx, error) {
 	tx.SetBytes(unsignedBytes, signedBytes)
 	return tx, nil
 }
-
-func initializeTx(cm codec.Manager, tx *Tx) error {
-	signedBytes, err := cm.Marshal(CodecVersion, tx)
-	if err != nil {
-		return fmt.Errorf("problem creating transaction: %w", err)
-	}
-
-	unsignedBytesLen, err := cm.Size(CodecVersion, &tx.Unsigned)
-	if err != nil {
-		return fmt.Errorf("couldn't calculate UnsignedTx marshal length: %w", err)
-	}
-
-	unsignedBytes := signedBytes[:unsignedBytesLen]
-	tx.SetBytes(unsignedBytes, signedBytes)
-	return nil
-}
```

Comment:
This is 100% duplicated from `tx.Initialize`. I have no recollection as to why this was done... but something seems to have been a mistake imo.